src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,51 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
```

Comment:
The `alive` array is used to track living animals, but it is not updated when an animal's health changes from below 100 to 100. Consider adding logic to handle such cases if required by the task.